packages/synapse-react-client/src/components/DynamicForm/DynamicForm.tsx

```diff
@@ -12,6 +12,7 @@ export type DynamicFormProps = {
   postUrl: string
   mutateFormDataBeforePost?: (formData: any) => any // allow caller to embed formdata into custom request body object
   onCancel?: () => void
+  onCustomSuccess?: () => void
```

Comment:
would "onSubmit" be more appropriate in this context?

Reply:
That sounds a lot better!